recipe/meta.yaml

```diff
@@ -1,48 +1,53 @@
 {% set name = "category_encoders" %}
-{% set version = "2.6.0" %}
-{% set sha256 = "bd9cfe8430b6cc2d67eaa51000937983422ebd81f88d770c3dcbe6cf9563e96e" %}
+{% set version = "2.6.1" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 51d4c594cd102a3d30a4a00f3ffa62f73b3d2f4298e7a57df8a8046a8d7881f5
+  patches:
+    - patches/replace_pkg_resources_with_importlib.patch
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script:
-    - {{ PYTHON }} -m pip install --no-deps -vvv .
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
     - python
     - pip
     - wheel
     - setuptools
   run:
     - python
-    - numpy >=1.14.0
     - scikit-learn >=0.20.0
     - scipy >=1.0.0
-    - statsmodels >=0.9.0
-    - pandas >=1.0.5
+    - pandas >=1.5.3
     - patsy >=0.5.1
+    - statsmodels >=0.9.0
+    - numpy >=1.20.0
+    - importlib_resources # [py<39]
 
 test:
   source_files:
     - tests
   requires:
     - pip
     - pytest
+    - pytest-cov
```

Comment:
You can remove this, as it's not being used anywhere.

Reply:
Great removed! And added clarifying comments for the skipped test.